### ISSUE TITLE
Please update the version that supports gnome 3.38 to extensions.gnome.org and debian repository

### ISSUE THREAD

**Issue by theoryfate** [micheleg/dash-to-dock#1317]
I know that gnome 3.38 is already supported, https://github.com/micheleg/dash-to-dock/pull/1309

But now we can only install git clone, but this cannot be managed by apt. 

Many people only download extensions from extensions.gnome.org, which will cause inconvenience.

**Comment by rnoons** [comment]
It's clearly not ready yet. The changes were only merged into master and not tagged yet for a release. They might be held up on testing or something.

**Comment by micheleg** [comment]
Sorry for the delay. Version tagged (71abe800b2bc0bca60c63c8e646db2a8a9827ddf) and uploaded to the extensions website (still in review). Finally migrated to 3.38 myself ...

**Comment by krassowski** [comment]
FYI, the new release breaks the extension on 3.36. See the comments on https://extensions.gnome.org/extension/307/dash-to-dock/.
